pkg/types/3bot_transactions.go

```diff
@@ -0,0 +1,2053 @@
+package types
+
+import (
+	"crypto/rand"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"math/big"
+
+	"github.com/threefoldtech/rivine/crypto"
+	"github.com/threefoldtech/rivine/pkg/encoding/rivbin"
+	"github.com/threefoldtech/rivine/types"
+)
+
```

Comment:
Also move the 3Bot Version constants to this file, as well as the specifiers and errors. These currently still are in the `transactions.go` file.